Verslag/methodology.tex

```diff
@@ -1,32 +1,71 @@
+Shodan.io was used for analyzing the IP addresses and domain names found with Wireshark and mitmproxy.
+Wireshark and mitmproxy were able to reveal all the network traffic sent and received by the application.
+Shodan.io revealed the other functions of the servers and whether there are similar servers, perhaps from the same hosting provider.
 
 \subsection{Code analysis}
 
-For the code analysis, JadX and Dex2Jar were used to decompile the application. This created the opportunity to read the code. An attempt was made to analyze the code. Analyzing the code was very difficult due to obfuscation. Where possible it was documented whether the code was malicious or not.
+For the code analysis, JadX and Dex2Jar were used to decompile the application.
+These applications turned the single .apk files into structured files, which were readable for the code analysis.
+This created the opportunity to read the code.
+An attempt was made to analyze the code.
+Analyzing the code was very difficult due to obfuscation.
+Where possible it was documented whether the code was malicious or not.
 
 \subsection{Process analysis}
-The Processes were analyzed to gain the insights that the research needs. The application had been started in the emulator of Android Studio. During the startup and use of the application, the Memory profiler had been used to monitor the memory usage of the application. All memory usage was documented and all peculiar behavior had been investigated further, to understand what had caused the suspicious behavior. Optionally, a heap dump was done during execution, which led to interesting findings.
+The Processes were analyzed to gain the insights that the research needs.
+The application had been started in the emulator of Android Studio.
+During the startup and use of the application, the Memory profiler had been used to monitor the memory usage of the application.
+All memory usage was documented and all peculiar behavior had been investigated further, to understand what had caused the suspicious behavior.
+Optionally, a heap dump was done during execution, which led to interesting findings.
 
 \subsection{Countermeasures and detection}
 
-To finalize the analysis, a YARA ruleset had been created for the application. All findings were documented into a report which was delivered to the team leader. These steps were performed by all the members. If an application needed additional steps this was declared at the start of the application chapter.
\ No newline at end of file
+To finalize the analysis, a YARA ruleset had been created for the application.
+And ways the user of the application can mitigate the effect of the malicious code. 
```

Comment:
Also, methods to mitigate the effect of the malicious code the end user could use were created